packages/desktop-gui/src/lib/utils.js

```diff
@@ -1,31 +1,7 @@
 import _ from 'lodash'
 import gravatar from 'gravatar'
+import duration from '../../../server/lib/util/duration'
```

Comment:
If it's code that needs to be shared between the server and desktop-gui, it should go in its own package, not be imported directly from `server`. I think we have some other places where this is done, but let's not add any more

Reply:
I think you originally wrote this, so I was just putting it back in since it was removed when we diverted from moment in the desktop-gui

Reply:
I don't think it's good to be willy-nilly requiring server files from the desktop-gui, since server files are written with the assumption that they'll be running in Node.js, not in the browser. I think it's ok for us to have our own shared `date-utils` or `shared-formatters` or w/e package for stuff like this.

Reply:
I was asking brian something else relevant to this PR and he said it shouldn't be necessary to create a package if it's only used once across packages - that we import stuff from other packages frequently. But that it should be imported as @package/server. I've updated the reference.

Reply:
I will register my dissent and resolve this conversation